sway/input/libinput.c

```diff
@@ -219,8 +219,9 @@ bool sway_input_configure_libinput_device(struct sway_input_device *input_device
 			ic->identifier, input_device->identifier);
 
 	bool changed = false;
-	if (ic->mapped_to_output &&
-			!output_by_name_or_id(ic->mapped_to_output)) {
+	if (strcmp("*", ic->mapped_to_output) != 0 &&
+		ic->mapped_to_output &&
```

Comment:
The `ic->mapped_to_output` check must come first, to avoid NULL pointer dereference.